<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProductSettings</class>
 <widget class="QDialog" name="ProductSettings">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>589</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>530</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>530</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
	background-color: rgb(191,193,194);
}

/*	Push button*/
QPushButton {
	min-height: 20px;
	border: 1px solid gray;
	border-radius: 3px;
	background-color: lightgray;
	selection-background-color: gray;
}

QPushButton:hover {
	background-color: gray;
	border-radius: 3px;
}

/*Combo box*/
QComboBox {
	min-height: 19px;
	background-color: lightgray;
	selection-background-color: gray;
	padding-left: 4px;
	border: 1px solid gray;
}

QComboBox:editable {
	min-height: 19px;
	background-color: lightgray;
	selection-background-color: gray;
	padding-left: 0px;
	border: 1px solid gray;	
}

QComboBox::down-arrow {
	image: url(C:/Projekti/arrow_down.png);
    width: 10px;
    height: 10px;
}

QComboBox::drop-down {
	border: 0px;
    subcontrol-position: top right;
    width: 15px;
}

QComboBox QAbstractItemView {
	border: 1px solid black;
	background-color: lightgray;
	selection-background-color: gray;
}


 QComboBox:hover {
	background-color: gray;
}


/*Line edit*/
QLineEdit,QTextEdit {
    border: 1px solid gray;
	border-radius: 3px;
    background-color: lightgray;
    selection-background-color: darkgray;
}

/*Tool button*/
QToolButton {
	max-width: 20px;
	max-height: 20px;
	border: 0px solid gray;
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>45</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(105,105,105)</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Product Settings</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButton">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../../../../Resources/Icons/close-icon.png</normaloff>../../../../Resources/Icons/close-icon.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea_2">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>-272</y>
        <width>516</width>
        <height>816</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
       </property>
       <property name="horizontalSpacing">
        <number>6</number>
       </property>
       <property name="verticalSpacing">
        <number>15</number>
       </property>
       <property name="leftMargin">
        <number>10</number>
       </property>
       <property name="topMargin">
        <number>10</number>
       </property>
       <property name="rightMargin">
        <number>10</number>
       </property>
       <property name="bottomMargin">
        <number>10</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label_22">
         <property name="text">
          <string>Product Category:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="m_pComboBoxProductCategory">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>21</height>
          </size>
         </property>
         <property name="editable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_23">
         <property name="text">
          <string>Product Name:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="m_pLineEditProductName"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_24">
         <property name="text">
          <string>Product Code:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="m_pLineEditProductCode"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_25">
         <property name="text">
          <string>Product Description:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QTextEdit" name="m_pTextEditProductDescription">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>50</height>
          </size>
         </property>
         <property name="tabStopWidth">
          <number>80</number>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_26">
         <property name="text">
          <string>Short Description:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="m_pLineEditShortDescription"/>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_27">
         <property name="text">
          <string>Product's URL:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLineEdit" name="m_pLineEditProductURL"/>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_28">
         <property name="text">
          <string>Producer:</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QComboBox" name="m_pComboBoxProducer">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>21</height>
          </size>
         </property>
         <property name="editable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_29">
         <property name="text">
          <string>Producer's Address:</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QTextEdit" name="m_pTextEditProducerAddress">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>50</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Producer's URL:</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QComboBox" name="m_pComboBoxProducerURL">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>21</height>
          </size>
         </property>
         <property name="editable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="label_30">
         <property name="text">
          <string>Place of Origin:</string>
         </property>
        </widget>
       </item>
       <item row="9" column="1">
        <widget class="QComboBox" name="m_pComboBoxPlaceOfOrigin">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>21</height>
          </size>
         </property>
         <property name="editable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="10" column="0">
        <widget class="QLabel" name="label_31">
         <property name="text">
          <string>Image:</string>
         </property>
        </widget>
       </item>
       <item row="10" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,1">
         <property name="spacing">
          <number>15</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="m_pLabelImage">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QFormLayout" name="formLayout_2">
           <property name="fieldGrowthPolicy">
            <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
           </property>
           <property name="horizontalSpacing">
            <number>10</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>Select:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QComboBox" name="m_pComboBoxImageSelect">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>21</height>
              </size>
             </property>
             <property name="editable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>Add New:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QPushButton" name="m_pPushButtonImageSelectFile">
             <property name="maximumSize">
              <size>
               <width>80</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>Select File...</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="11" column="1">
        <widget class="QFrame" name="frame">
         <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="0,0,5,4">
          <property name="spacing">
           <number>10</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QComboBox" name="m_pComboBoxProductUnit">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>21</height>
             </size>
            </property>
            <property name="editable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_20">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Unit Quantity:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="m_pComboBoxUnitQuantity">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>21</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>90</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="editable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item row="12" column="0">
        <widget class="QLabel" name="label_35">
         <property name="text">
          <string>Tax Rate:</string>
         </property>
        </widget>
       </item>
       <item row="12" column="1">
        <widget class="QComboBox" name="m_pComboBoxTaxRate">
         <property name="maximumSize">
          <size>
           <width>70</width>
           <height>16777215</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>9.5%</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>22%</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="14" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Location:</string>
         </property>
        </widget>
       </item>
       <item row="14" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,0,0,0,0,0,0,0">
         <property name="spacing">
          <number>2</number>
         </property>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="text">
            <string>X:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="m_pLineEditLocationX">
           <property name="text">
            <string>0.0</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="text">
            <string>Y:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="m_pLineEditLocationY">
           <property name="text">
            <string>0.0</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="text">
            <string>Z:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="m_pLineEditLocationZ">
           <property name="text">
            <string>0.0</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="15" column="0">
        <widget class="QLabel" name="label_33">
         <property name="text">
          <string>Quantity:</string>
         </property>
        </widget>
       </item>
       <item row="15" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="0,0,0,0,0">
         <property name="spacing">
          <number>10</number>
         </property>
         <item>
          <widget class="QLineEdit" name="m_pLineEditQuantity">
           <property name="minimumSize">
            <size>
             <width>70</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>70</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_19">
           <property name="text">
            <string>Last Changed:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="m_pLineEditQuantLastChanged">
           <property name="minimumSize">
            <size>
             <width>70</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_6">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="17" column="0">
        <widget class="QLabel" name="label_34">
         <property name="text">
          <string>Price:</string>
         </property>
        </widget>
       </item>
       <item row="17" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <property name="spacing">
          <number>10</number>
         </property>
         <item>
          <widget class="QLineEdit" name="m_pLineEditPrice">
           <property name="minimumSize">
            <size>
             <width>70</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>70</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_17">
           <property name="text">
            <string>Last Changed:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="m_pLineEditPriceLastChanged">
           <property name="minimumSize">
            <size>
             <width>70</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_7">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="11" column="0">
        <widget class="QLabel" name="label_32">
         <property name="text">
          <string>Product Unit:</string>
         </property>
        </widget>
       </item>
       <item row="21" column="0" colspan="2">
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <property name="spacing">
          <number>10</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="m_pPushButtonSave">
           <property name="text">
            <string>Save</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="m_pPushButtonCancel">
           <property name="text">
            <string>Cancel</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="18" column="0" colspan="2">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>toolButton</sender>
   <signal>clicked(bool)</signal>
   <receiver>ProductSettings</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>507</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>264</x>
     <y>294</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pPushButtonCancel</sender>
   <signal>clicked(bool)</signal>
   <receiver>ProductSettings</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>383</x>
     <y>566</y>
    </hint>
    <hint type="destinationlabel">
     <x>264</x>
     <y>294</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
